devops/repos/create_pkgs.sh

```diff
@@ -10,15 +10,15 @@ cd $HIPSYCL_PKG_SCRIPT_DIR_ABS/packaging
 
 stage_dir=$HIPSYCL_PKG_DEVOPS_DIR/$HIPSYCL_PKG_UBUNTU_PKG_DIR
 echo "Starting Ubuntu 18.04 pkg build"
-singularity exec $HIPSYCL_PKG_CONTAINER_DIR/hipsycl-ubuntu-18.04.sif sh make-ubuntu-pkg.sh
+singularity exec $HIPSYCL_PKG_CONTAINER_DIR/hipsycl-ubuntu-18.04 sh make-ubuntu-pkg.sh
```

Comment:
By the way.. Do you have any opinion regarding Ubuntu 20.04 and whether/for how long we should still support 18.04?

Reply:
Not really, but with this update to the packages it should be fairly simple to add support for 20.04, (and for other oses as well)